### PR TITLE
Publish GridFTP and GSI transition plan

### PR DESCRIPTION
**Pull request by brianhlin**
Frank said we need to publish this and link it from www.opensciencegrid.org before the council meeting on Dec 13. This is just a combination of the intro + FAQ that we put together so I'm open to any changes if we want to take this in another direction.

### CODE REVIEW

File: docs/policy/gridftp-gsi-migration.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+| Jan 2021  | OSG series 3.6, without GCT dependencies, is released.                                                 |               |
+| Jan 2022  | End of support for OSG 3.5.                                                                            |               |
+
+Frequently Asked Questions
+--------------------------
+
+### How does SciTokens interoperate with other token technologies in the WLCG? ###
+
+We have been very active with the WLCG AuthZ working group and the WLCG JWT profile incorporates many core pieces of
+SciTokens -- enough that we feel confident in saying that we will support the relevant parts of WLCG JWT in the
```
Comment (bbockelm) [inline docs/policy/gridftp-gsi-migration.md]: A bit informal.  Given 0.4.0 came out in the meantime, perhaps:

> The `scitokens-cpp` library used by OSG can transparently use both WLCG JWTs and SciTokens.

File: docs/policy/gridftp-gsi-migration.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+SciTokens client library for existing adopters.
+
+### Will a US-LHC migration from GridFTP to XRootD require the same migration for WLCG? ###
+
+Europe has been an enthusiastic partner in doing this switch.
+That said, there are a number of options to disentangle the two if the pace of evolution turns out to be very different.
```
Comment (bbockelm) [inline docs/policy/gridftp-gsi-migration.md]: We have been working to ensure to coordinate our activities through the WLCG DOMA group.

File: docs/policy/gridftp-gsi-migration.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+### What role does LCMAPS play with SciTokens? ###
+
+LCMAPS only works with GSI.
+The model for SciTokens is sufficiently more simple for sites that the full complexity of LCMAPS is a bit overkill.
```
Comment (bbockelm) [inline docs/policy/gridftp-gsi-migration.md]:
```suggestion
The model for SciTokens is sufficiently more simple for sites that the full complexity of LCMAPS is not needed.
```

File: docs/policy/gridftp-gsi-migration.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+### What does a SciTokens transition for GlideinWMS mean for European sites? ###
+
+The only piece that involves European sites is the factory to CE relationship:
+given HTCondor-CE is covered, we need to focus on ARC-CE.
```
Comment (bbockelm) [inline docs/policy/gridftp-gsi-migration.md]:
```suggestion
given HTCondor-CE 4.0 already supports SciTokens, we have begun to engage with the ARC-CE team.
```

File: docs/policy/gridftp-gsi-migration.md (hunk 2 above)
Comment (brianhlin) [reply docs/policy/gridftp-gsi-migration.md]: It's phrased as a yes/no question so I've changed this answer to:

> No but we have been working to ensure that we coordinate our activities through the WLCG DOMA group.

**Pull request merged**